lib/stormpath-sdk/jwt/authentication_result.rb

```diff
@@ -0,0 +1,7 @@
+module Stormpath
+  module Jwt
```

Comment:
Yea, I got the idea from here https://github.com/stormpath/stormpath-sdk-node/tree/master/lib/jwt

I agree, would move it to Oauth module. having in mind discussion on changes above we also need to change where is this method called.
should i move it to through application
```ruby
application.verify_access_token token
```
or maybe do something like this
```ruby
Stormpath::Oauth::VerifyAccessToken.new(application).verify(authorization_token)
```